src/ctapipe/reco/stereo_combination.py

```diff
@@ -405,3 +415,251 @@ def predict_table(self, mono_predictions: Table) -> Table:
         stereo_table[f"{self.prefix}_telescopes"] = tel_ids
         add_defaults_and_meta(stereo_table, _containers[self.property], self.prefix)
         return stereo_table
+
+
+class StereoDispCombiner(StereoCombiner):
+    """ """
+
+    weights = CaselessStrEnum(
+        ["none", "intensity", "konrad"],
+        default_value="none",
+        help=(
+            "What kind of weights to use. Options: ``none``, ``intensity``, ``konrad``."
+        ),
+    ).tag(config=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.supported = ReconstructionProperty.GEOMETRY
+        if self.property not in self.supported:
+            raise NotImplementedError(
+                f"Combination of {self.property} not implemented in {self.__class__.__name__}"
+            )
+
+    def _calculate_weights(self, data):
+        if isinstance(data, Container):
+            if self.weights == "intensity":
+                return data.hillas.intensity
+
+            if self.weights == "konrad":
```

Comment:
I think I've made this comment before: can we pick a better name here, since calling it "konrad" is not very helpful?  This weighting is  _intensity_  x _aspect-ratio_, so we could call it "intensity-aspect-ratio" or "aspect-weighted-intensity" or maybe there is a shorter word...